### PR TITLE
Add Landing page content

### PR DESCRIPTION
**Pull request by phattya**
# Pull Request Template

## Issue Overview
Add Landing Page Content and ES translations

This PR addresses #issue

- [ ] This change addresses the issue in full
- [x] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description
Add Landing page content from team shown in [this sheet](https://docs.google.com/spreadsheets/d/1Cr3L2H2iLWMS1qNOYYsnLJCrB1RNo_PXQiirzfg38PE/).  The language is also shown in the Figma Landing page mocks, but the source of truth is the sheet.

We've only received ES & ZH translations back so far, so going to submit this PR now to unblock progress. 
image
image

Still need to:
-  add 2 other language translations

## How Can This Be Tested/Reviewed?
`yarn dev` on this branch and look at the Landing page. Test that the links are working and that all the content looks similar to the Figma mocks (or the screenshots included here). Switch languages and ensure no translations are missing (ES & ZH only)
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by phattya** [comment]
I'm guessing I introduced this error in this code ... I haven't hand time to investigate yet, so feel free to make suggestions ...
image

**Review by ifranch — APPROVED** [review]

File: shared-helpers/src/locales/es.json [hunk 1]
```diff
@@ -640,7 +641,8 @@
   "pageTitle.getAssistance": "Obtener asistencia",
   "pageTitle.housingCounselors": "Asesores de vivienda",
   "pageTitle.privacy": "Política de privacidad",
-  "pageTitle.rent": "Vivienda de alquiler razonable",
+  "pageTitle.rent": "Vivienda de aliquiler razonable",
```
[the remark this answers is not in the document]
Comment (phattya) [reply shared-helpers/src/locales/es.json:644]: Ha! I was like how did that even change? Realize I had hit an `i` to edit in vim and didn't catch it at the moment. Great catch!

File: shared-helpers/src/locales/general.json [hunk 2]
```diff
@@ -1011,9 +1011,22 @@
   "users.userUpdated": "User updated",
   "welcome.allApplicationClosed": "All applications are currently closed, but you can view closed listings.",
   "welcome.introduction": "Welcome to the Doorway Housing Portal",
-  "welcome.findAffordableHousing": "Find affordable housing in the Bay Area",
-  "welcome.useDoorway": "Use the Doorway Housing Portal to apply for affordable housing across the Bay Area. Getting housing is hard. Doorway is working to make it easier.",
+  "welcome.findAffordableHousing": "Find and apply for affordable housing in the Bay Area",
+  "welcome.useDoorway": "The portal provides a centrial place for housing seekers to find and apply for affordable housing throughout the Bay Area.",
```
[the remark this answers is not in the document]
Comment (phattya) [reply shared-helpers/src/locales/general.json:1015]: guess I missed it here too. Great catch!

**Comment by phattya** [comment]
> I'm guessing I introduced this error in this code ... I haven't hand time to investigate yet, so feel free to make suggestions ... image

Fixed this in the last commit.

**Review by lisunshiny — APPROVED** [review]

**Pull request merged**